Library/Formula/assh.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Assh < Formula
+  desc "assh: Advanced SSH config - A transparent wrapper that adds regex, aliases, gateways, includes, dynamic hostnames to SSH"
+  homepage "https://github.com/moul/advanced-ssh-config"
+  url "https://github.com/moul/advanced-ssh-config/archive/v2.2.0.tar.gz"
+  sha256 "8a71969df06e8db0bd4b06cf928780e45e22cf5f131f06aec43cccd955ee2ead"
+
+  head "https://github.com/moul/advanced-ssh-config.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GOBIN"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+    (buildpath/"src/github.com/moul/advanced-ssh-config").install Dir["*"]
+
+    system "go", "build", "-o", "#{bin}/assh", "-v", "github.com/moul/advanced-ssh-config/cmd/assh/"
+  end
+
+  test do
+    output = shell_output(bin/"assh --version")
```

Comment:
Ok, considering the failures may come from:
- the build (already tested)
- the configuration file (not relevant in a sandbox)
- testing the connectivity with a real SSH server (may include false-positive depending on host)

I think we have enough tests for now

I've open an [issue](https://github.com/moul/advanced-ssh-config/issues/108) so we can discuss improvements for the next releases